tests/shared/src/main/scala/catseffect/examples.scala

```diff
@@ -83,10 +83,12 @@ package examples {
       } catch {
         case _: TimeoutException => println("sadness (timeout)")
       }
-      Thread.sleep(500L)
-      // by now the WSTP (and all its threads) must've been shut down:
       if (thread eq null) println("sadness (thread is null)")
-      else if (thread.isAlive()) println("sadness (thread is alive)")
+      else {
+        thread.join(2000)
```

Comment:
```suggestion
        thread.join(2000L)
```

(I'm surprised we don't have a warning for this. Not that it matters all that much...)